File: backend/clear_users_only.py
import sys
import os

sys.path.append(os.path.dirname(os.path.abspath(__file__)))

from app import create_app
from models import User, Profile, PendingUser

def clear_users():
    app = create_app()
    with app.app_context():
        # Delete all documents in these collections
        User.objects.delete()
        Profile.objects.delete()
        PendingUser.objects.delete()
        print("Successfully deleted all Users, Profiles, and Pending registrations.")

if __name__ == "__main__":
    clear_users()

File: backend/src/scripts/clearUsersOnly.js
/**
 * scripts/clearUsersOnly.js
 * Deletes all Users, Profiles, and PendingUsers.
 */

import dotenv from 'dotenv';
import connectDB from '../config/db.js';
import { User, Profile, PendingUser } from '../models/index.js';
import mongoose from 'mongoose';

dotenv.config();

async function main() {
  await connectDB();
  await User.deleteMany();
  await Profile.deleteMany();
  await PendingUser.deleteMany();
  console.log('Successfully deleted all Users, Profiles, and Pending registrations.');
  await mongoose.disconnect();
}

main().catch((err) => {
  console.error(err);
  process.exit(1);
});
